### PR TITLE
implemented Session.transpose

### PR DESCRIPTION
**Pull request by gdementen**

### CODE REVIEW

File: doc/source/changes/version_0_25.rst.inc [hunk 1]
```diff
@@ -72,6 +72,36 @@
...
+    >>> sess = Session([('arr1', arr1), ('arr2', arr2)])
+    >>> def print_summary(s):
+    ...     print(s.summary("{name} -> {axes_names}"))
+    >>> print_summary(sess)
+    arr1 -> a, b, c
+    arr2 -> a, b
+
+  Transpose axes of all arrays
+
+    >>> print_summary(sess.transpose('b', 'a'))
```
Comment (alixdamman) [inline doc/source/changes/version_0_25.rst.inc]: Why not just sess.transpose('b') ? 
Given the examples, I fear people will think that transpose works like swap (people not used to Numpy).

File: larray/core/session.py [hunk 2]
```diff
@@ -688,6 +688,58 @@ def __eq__(self, other):
...
+        >>> sess = Session([('arr1', arr1), ('arr2', arr2)])
+        >>> def print_summary(s):
+        ...     print(s.summary("{name} -> {axes_names}"))
+        >>> print_summary(sess)
+        arr1 -> a, b, c
+        arr2 -> a, b
+
+        Transpose axes of all arrays
+
+        >>> print_summary(sess.transpose('b', 'a'))
```
Comment (alixdamman) [inline larray/core/session.py]: same as in changelog.

File: larray/core/session.py [hunk 3]
```diff
@@ -688,6 +688,58 @@ def __eq__(self, other):
...
+        ----------
+        *args
+            Accepts either a tuple of axes specs or axes specs as `*args`. Omitted axes keep their order.
+            Use ... to avoid specifying intermediate axes. Axes missing in an array are ignored.
+
+        Returns
+        -------
+        Session
+            Session with each array with reordered axes where appropriate.
+
```
Comment (alixdamman) [inline larray/core/session.py]: See Also
LArray.transpose

**Comment by gdementen** [comment]
@alixdamman followed your suggestions. Ok to merge?

**Comment by alixdamman** [comment]
Yes

**Pull request merged**